src/Helpers/BuildUri.php

```diff
@@ -17,7 +17,7 @@ public function execute(string $path, array $params = []): string
 
         foreach ($params as $key => $value) {
             if (is_array($value)) {
-                $value = implode(',', $value);
+                $value = implode("&$key=", $value);
             }
 
             $paramsArray[] = $key.'='.$value;
```

Comment:
This does not work, it will end up generating `tags=dolorem&$key=lorem`,  assuming I pass ['dolorem', 'lorem']. tags should be an array.